sotodlib/io/bookbinder2.py

```diff
@@ -0,0 +1,931 @@
+import so3g
```

Comment:
I'm a bit hesitant to start adding in "bookbinder2." It means we'll have some reason to start having 3, 4, 5, etc in the future. I know this is a massive change to the codebase and we want to acknowledge the previous work, particularly by @algadoreo. Especially since we wouldn't have this version without having gone through making the first version and figuring out all the hard parts and interfaces. But this is all version controlled so we will still have the history.

Reply:
haha yes totally. Its convenient right now to have both this and `bookbinder.py` in the codebase for the sake of comparison, but I do plan on renaming this to `bookbinder.py` and replacing the old version before merge.